apphosting.mainnet.yaml

```
@@ -23,6 +23,8 @@ env:
     value: https://api.like.co
   - variable: LIKECOIN_CHAIN_API_ENDPOINT
     value: https://mainnet-node.like.co
+  - variable: LIKECOIN_EVM_CHAIN_API_ENDPOINT
+    value: https://likenft-indexer.pandawork.com/api
```

```suggestion
    value:
```
Better leave empty until we have mainnet indexer